dvc/command/status.py

```diff
@@ -10,6 +12,10 @@ class CmdDataStatus(CmdDataBase):
     STATUS_LEN = 20
     STATUS_INDENT = "\t"
     UP_TO_DATE_MSG = "Data and pipelines are up to date."
+    EMPTY_PROJECT_MSG = (
```

Comment:
Use `format_link`?
https://github.com/iterative/dvc/blob/46bc89319a23a114c26c1e2d4020a436b2730be0/dvc/utils/__init__.py#L382

Reply:
Can't argue with that. Fixed in 656efd56626ca5edeab62d60020619b8a6b798ae.